### DISCUSSION TITLE
Reading URL

### DISCUSSION THREAD

**Discussion by erasmushaggblom** [pygame-web/pygbag#122] [Q&A] [answered]
Hi all,

As a follow-up to my previous issue (https://github.com/pygame-web/pygbag/discussions/120), when my game is deployed, the players will access it from a URL containing a unique identifier (eg. <main_url>?id=232jaklzo). I need to be able to read the URL from the game to make sure that the identifier is valid. I have not been able to figure out how to do so, is it feasible?

**Comment by pmp-p** [comment, marked as answer]
you will find "id=232jaklzo" in PyConfig.orig_argv,  sys.argv is after the # not the ?

eg
[?cpython311&-i&-X&dev#main.py 0 1 2 3](https://pygame-web.github.io/showroom/pythongit.html?cpython311&-i&-X&dev#main.py%200%201%202%203) will give

```
>>> 
>>> PyConfig.orig_argv
['cpython311', '-i', '-X', 'dev']
>>> sys.argv
['main.py', '0', '1', '2', '3']
>>> 
```

**Reply by erasmushaggblom** [reply]
Thank you very much!
